rudder-agent/SOURCES/patches/fusioninventory/13964-addOSReleaseSupport.patch

```diff
@@ -0,0 +1,76 @@
+diff --git a/lib/FusionInventory/Agent/Task/Inventory/Linux/Distro/LSB.pm b/lib/FusionInventory/Agent/Task/Inventory/Linux/Distro/LSB.pm
+index 44d26ec..4415a37 100644
+--- a/lib/FusionInventory/Agent/Task/Inventory/Linux/Distro/LSB.pm
++++ b/lib/FusionInventory/Agent/Task/Inventory/Linux/Distro/LSB.pm
+@@ -5,8 +5,11 @@ use warnings;
+ 
+ use FusionInventory::Agent::Tools;
+ 
++our $runMeIfTheseChecksFailed =
```

Comment:
In Agent::Task::Inventory, line 219.
Since LSB.pm is no longer the "classic" scenario executed, It should only be executed if OSRelease is not available.